### PR TITLE
create change-case extension

### PR DESCRIPTION
**Pull request by es183923**
## Description

create a change case extension
It is a wrapper for [`change-case`](https://www.npmjs.com/package/change-case).
Inspired by [`vscode-change-case`](https://github.com/wmaurer/vscode-change-case)
redo of #80
## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/72744903/137641165-d53a32c2-60bc-42f4-8f9f-d5fadb766fde.mp4

## Checklist

- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by tdwesten** [comment]
Wow i Like this one!

File: extensions/change-case/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "change-case",
+  "title": "Change Case",
+  "description": "Change the text in the clipboard",
```
Comment (mattisssa) [inline extensions/change-case/package.json:4]: I would be more specific here and propose something like this:
```suggestion
  "description": "Transform a string between camelCase, PascalCase, Capital Case, snake_case, param-case, CONSTANT_CASE and others",
```

File: extensions/change-case/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "icon": "command-icon.png",
+  "author": "es183923",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "change-case",
+      "title": "Change Case",
+      "subtitle": "",
+      "description": "Change the text in the clipboard",
```
Comment (mattisssa) [inline extensions/change-case/package.json:13]:
```suggestion
      "description": "Changes a text in the clipboard",
```

**Comment by mattisssa** [comment]
Hey @es183923, have you changed your `author` field in the `package.json`? Seems like we have some troubles with publishing your extension. Investigating what could it be

**Comment by fturcheti** [comment]
Hey @es183923! This is a really handy extension, but I'm having trouble with it because `change-case` doesn't support diacritics (https://github.com/blakeembrey/change-case/issues/138#issuecomment-733239699) and I need them because portuguese is my first language.

I found this other package, [case-anything](https://www.npmjs.com/package/case-anything), that has proper support for diacritics. What do you think about changing `change-case` for `case-anything` in your extension?

**Pull request merged**